src/App.js

```diff
@@ -30,35 +31,35 @@ class App extends Component {
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    // const testItem = this.spawnItem(Date.now());
+    // this.state.items.push(testItem);
 
-    // Uncomment this to automatically spawn new items
     this.enableSpawner();
-
-    console.log(this.state);
   }
-
+  
   onItemClicked = () => {
-    // Fill this in!
+    this.setState(prevState => {
```

Comment:
Nice! I love the `prevState` syntax. It's a little different than what we did in class, but I like it a lot-- well done!